charts/cofide-observer/values.yaml

```diff
@@ -0,0 +1,90 @@
+# Default values for cofide-observer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+observer:
+  connectURL:
+  connectTrustDomain:
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+# This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
+image:
+  registry: 010438484483.dkr.ecr.eu-west-1.amazonaws.com
+  repository: cofide/cofide-observer
+  # This sets the pull policy for images.
+  pullPolicy: IfNotPresent
```

Comment:
Minor but we use `Always` throughout the other charts. Maybe that needs to change though?

Reply:
IfNotPresent is the default generated by Helm. I think the main reason to use Always would be if your tags were mutable (e.g. latest), then you might need to always pull an image to get the most up to date one. Is there another reason we might want Always?

Reply:
Probably not, we can move the others back to IfNotPresent